src/Search.php

```diff
@@ -6211,7 +6211,24 @@ public static function displayConfigItem($itemtype, $ID, $data = [])
                     $out = " class=\"shadow-none\" style=\"background-color: #cf9b9b\" ";
                 }
                 break;
-
+            case "glpi_certificates.date_expiration":
+                if (
```

Comment:
`"glpi_certificates.date_expiration":` was already in the function (at the end), so now it is there twice

Reply:
refactored